packages/@sanity/code-input/src/CodeInput.js

```diff
@@ -64,6 +67,31 @@ export default class Code extends React.PureComponent {
     ]))
   }
 
+  handleToggleSelectLine = line => {
+    const {type, onChange} = this.props
+    const path = ['highlightedLines']
+
+    const highlightedLines = xor(this.props.value.highlightedLines, [line]).sort(compareNumbers)
+
+    onChange(PatchEvent.from([
+      setIfMissing({_type: type.name, highlightedLines: []}),
+      line ? set(highlightedLines, path) : unset(path)
```

Comment:
To answer my own question: No. But `line` is lineNumber, and that can never be falsey AFAIK. I suggest renaming it to `lineNumber` for clarity.